assessment.sql

```diff
@@ -5,24 +5,58 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
+CREATE TABLE licencees(
+  P_Id int NOT NULL,
```

Comment:
This sets an integer column where `NULL` is not allowed, but it will not set the ID value for you. You will be responsible for setting the `P_id` for each row as you insert it. This effectively makes your `INSERT` and `\copy` statements that follow incorrect.